XCUITests/TopTabsTest.swift

```diff
@@ -63,7 +63,7 @@ class TopTabsTest: BaseTestCase {
         waitforExistence(app.webViews.links.staticTexts["More information..."])
         app.webViews.links.staticTexts["More information..."].press(forDuration: 5)
         app.buttons["Open in New Tab"].tap()
-        app.buttons["Switch"].tap()
+        sleep(2)
```

Comment:
The thing here is that the Switch button is in a banner that last for a few seconds, not sure if wait for that element will make the test always consistent, because if the timing is not good, if we wait once it has appeared and dissapeared, then the test will fail...
Will try that anyway and see if the test passes. Locally it does not fail, but running in bb it was not able to tap on the button, although looking at the screenshots logs, the button was there...